assembl/templates/system/system_user_information.jinja2

```diff
@@ -0,0 +1,5 @@
+hostname = {{ hostname }}
```

Comment:
This template is only used in the fabfile in a function called `create_user`. This function used to create a user and take care of all the .ssh configuration. Since you handled that in the scripts, I don't think we need that function anymore and hence we don't need this template

Reply:
I'll be removing these. They're outdated code from the `add_deploy_job` branch, which was the first go at CICD. The assumptions there are no longer relevant.